java/arcs/core/storage/database/DatabaseManager.kt

```diff
@@ -56,6 +56,15 @@ interface DatabaseManager {
 
     /** Garbage collection run: removes unused entities. */
     suspend fun runGarbageCollection()
+
+    /** Gets the count of entities stored. */
```

Comment:
/** Gets the sum of the number of entities stored across all databases. */

Reply:
Done.